implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -447,3 +449,78 @@ pub(crate) fn replace_hyphen_with_stdin(s: String) -> String {
         s
     }
 }
+
+const ENV_FORCE_COLOR: &str = "ockam_force_color";
+
+fn show_help(help: clap::Error) {
+    use std::env;
+    let mut try_fallback = false;
+    let preferred_pager = env::var_os("PAGER").unwrap_or_else(|| {
+        try_fallback = true;
+        OsString::from("less")
+    });
+
+    if preferred_pager == "false" {
+        use clap::{ColorChoice::*, CommandFactory};
+        let possibly_forced = if env::var_os(ENV_FORCE_COLOR).is_some() {
+            Always
+        } else {
+            Auto
+        };
+
+        help.with_cmd(&OckamCommand::command().color(possibly_forced))
+            .exit();
+    }
```

Comment:
The whole function feels obscure and with lots of hidden subtleties around the pager parsing. Here are some questions that I had while reading this particular block:

- This feels hacky, why not use an Option to represent "false" instead?
- If a user sets the "PAGER" env var to "false", why would they want to set it in the first place? I'd rather provide another env var to disable this feature.
- The user could have set the `NO_COLOR` env var, which is not taken into consideration here
- What's happening in this block? I had to read the documentation of the `with_cmd` and `exit` commands to learn that it's an early exit of the process. I also had to get into the details of the `paginate_with` to understand that they depend on each other. I wonder if this block should live in the `paginate_with` function.